scene/resources/visual_shader.h

```diff
@@ -371,10 +371,10 @@ class VisualShaderNodeCustom : public VisualShaderNode {
 	GDVIRTUAL0RC(String, _get_category)
 	GDVIRTUAL0RC(int, _get_return_icon_type)
```

Comment:
Where is `_get_return_icon_type` used? I couldn't find a `GDVIRTUAL_CALL`. It seems like the return type should also be `PortType` but I'm not sure so I didn't change it.

Reply:
It's used here:

https://github.com/godotengine/godot/blob/d2d5e548c9cac48a89b97ed4dd9bfc93b4528f0d/editor/plugins/visual_shader_editor_plugin.cpp#L1244

Could possibly be changed to `GDVIRTUAL_CALL`?

Reply:
I don't think we can use the `GDVIRTUAL_CALL` macro here. The method is called on `p_custom_node` and is defined as `protected` in a different type so likely not accessible anyway.